src/Microsoft.Health.Dicom.Core/DicomCoreResource.Designer.cs

```diff
@@ -87,6 +87,15 @@ internal class DicomCoreResource {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to The custom tag &apos;{0}&apos; has already been supported..
+        /// </summary>
+        internal static string CustomTagAlreadySupported {
```

Comment:
>Supported [](start = 47, length = 9)

This should probably be already exists to match the other exceptions!

Reply:
Hi Sneha, could you point out the duplicated exception?

Reply:
Sorry, I mean the wording here for this exception should be CustomTagAlreadyExists instead of supported.

Reply:
Actually I'm not sure what's the right word to put here, maybe  you have better idea.

There are 2 case of "already exist":
1. The custom tag has been added before (currently it fail for DuplicatedCustomTag)
2. The custom tag is one of tag we supported now ( e.g: PatientName https://github.com/microsoft/dicom-server/blob/master/docs/resources/conformance-statement.md#searchable-attributes)

This error message is for #2. 
If give CustomTagAlreadyExists, would that be confused with #1?

Reply:
hmm not sure... I'm absolutely the worst with names. I think it sounds confusing to say it is already supported since it doesn't convey what you want. I would say RequestedTagAlreadyExists or something?